src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__kfwd_dashboard.sql

```diff
@@ -609,6 +665,20 @@ select
         else 0
     end as is_grad_ever,
 
+    case
+        when
+            ei.ugrad_status = 'Graduated'
+            or ei.cte_status = 'Graduated'
+            and if(
+                ei.ugrad_status = 'Graduated',
+                ei.ugrad_actual_end_date,
+                ei.cte_actual_end_date
+            )
+            <= date((c.ktc_cohort + 6), 08, 31)
+        then 1
```

Comment:
you should break this up into 2 different `when ... then` clauses. one for ugrad and one for cte

Reply:
oh duh - thanks